packages/dashboard-frontend/src/components/WorkspaceProgress/CreatingSteps/Fetch/Devfile/__tests__/index.spec.tsx

```diff
@@ -742,6 +743,46 @@ describe('Creating steps, fetching a devfile', () => {
       expect(mockOpenOAuthPage).not.toHaveBeenCalled();
       expect(mockOnError).not.toHaveBeenCalled();
     });
+
+    it('should show warning on SSH url', async () => {
+      const expectAlertItem = expect.objectContaining({
+        title: 'Warning',
+        variant: AlertVariant.warning,
+        children: (
+          <ExpandableWarning
+            textBefore="Devfile resolve from a privatre repositry via an SSH url is not supported."
+            errorMessage="Could not reach devfile"
+            textAfter="You can apply a Personal Access Token to be able to fetch the devfile.yaml content."
```

Comment:
if applying PAT solves the issue, I'd change the wording to this:

"Apply a Personal Access Token to fetch the devfile.yaml content."

Reply:
Thanks, done